unit_tests/test_aggregator.py

```diff
@@ -40,10 +34,8 @@ def mock_prefixes(mocker):
 
 @pytest.fixture(autouse=True)
 def mock_start_websocket_funcs(mocker):
-    """
-    Fixture mocks external classes used in Aggregator.start_websocket_connections
-    """
-    # mocker.patch("gsy_e_sdk.aggregator.AggregatorWebsocketMessageReceiver")
+    """Mock external classes used in Aggregator.start_websocket_connections."""
+    mocker.patch("gsy_e_sdk.aggregator.AggregatorWebsocketMessageReceiver")
```

Comment:
I patched the `AggregatorWebsocketMessageReceiver` object here and reverted the patch that was done in `mock_execute_batch_command_methods`

Reply:
You mean that you reverted it because it would then be redundant to have the class and a method of it patched as well, right?

Reply:
Exactly